Resources/Prototypes/_LostParadise/Entities/Objects/Specific/Robotic/borg_modules_syndi_assault.yml

```
@@ -0,0 +1,47 @@
+- type: entity
+  id: LPPBorgModuleAssaultRam
+  parent: [ BaseBorgModuleSyndicateAssault, BaseProviderBorgModule ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)